packages/slice-machine/server/src/api/screenshots/puppeteer.ts

```diff
@@ -39,12 +43,14 @@ export default {
 const generateScreenshot = async (
   browser: puppeteer.Browser,
   screenshotUrl: string,
-  pathToFile: string
+  pathToFile: string,
+  screenWidth: string
 ): Promise<void> => {
   // Create an incognito context to isolate screenshots.
   const context = await browser.createIncognitoBrowserContext();
   // Create a new page in the context.
   const page = await context.newPage();
+  await page.setViewport({ width: Number(screenWidth), height: 600 });
```

Comment:
The height shouldn't be hardcoded.
For now this feature allows the user to take a screenshot with only 3 possible values.
In the future we could image the user being able to pass any value through inputs or something like that, to provide more flexibility.
Even though the height is 600 for now, it should be controlled by the frontend entirely.
The frontend being the one to set the intention and parameters of the screenshot.

Reply:
I think the screenWidth property should be a dimension object.